releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -110,29 +108,35 @@ node evm/its.js --action registerTokenMetadata --tokenAddress [tokenAddress]
 - [ ]  Run from one chain to link to the remote token
 
 ```bash
-# Register source token
+# Register source token. Record tokenId from output for next steps. 
 node evm/interchainTokenFactory.js --action registerCustomToken --tokenAddress [tokenAddress] --tokenManagerType 4 --operator [wallet] --salt "salt6789"
 
 # Link to remote token. Ensure GMP call is executed
 node evm/interchainTokenFactory.js --action linkToken --destinationChain chain2 --destinationTokenAddress [remote token address] --tokenManagerType 4 --linkParams "0x" --salt "salt6789"
 ```
 
+- [ ] Fetch tokenManager address for deployed token on both chains
+
+> [!NOTE] 
+> Use `tokenId` from `registerCustomToken` tx output
+
+```bash
+# Record tokenManager address from output for transferMintership
+node evm/its.js --action tokenManagerAddress --tokenId [tokenId]
+```
+
 - [ ] Run on both chains
 
 ```bash
 # Transfer mintership for each token to the token manager
 node evm/its.js --action transferMintership --tokenAddress [tokenAddress] --minter [tokenManager]
 ```
 
-> [!NOTE]  
-> `tokenManager` address is different per token, which can be found on tx output from `registerCustomToken` action on chain
-
 - [ ]  Interchain Transfer (both ways)
 
+> [!NOTE] 
+> Use `tokenId` from `registerCustomToken` tx output
+
```

Comment:
```suggestion
```
Same. We want to aim for the docs to be as concise as possible for readability while being precise